### PR TITLE
Security feature working

### PR DESCRIPTION
**Pull request by snack-attack**

### CODE REVIEW

**Review by alicemcneil — COMMENTED** [review]
looks good! great job 👍

File: css/login.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+ /* Bordered form */
+ form {
+    border: 3px solid #f1f1f1;
+    width: 50%;
+    margin: 12vh auto;
+  }
+  
+  /* Full-width inputs */
```
Comment (sjr-mayden) [inline css/login.css]: Inconsistent comments, should be with or without leading spaces all the same

File: css/login.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    cursor: pointer;
+    width: 100%;
+  }
+  
+  /* Add a hover effect for buttons */
+  button:hover {
+    opacity: 0.8;
+  }
+  
+
```
Comment (sjr-mayden) [inline css/login.css]: Extra line break

File: login.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <link rel="stylesheet" href="./css/login.css">
+    </head>
+    <body>
+        <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+        <form method="POST" action="login.php">
+            <div class="container">
+              <label for="uname"><b>Username</b></label>
+              <input type="text" placeholder="Enter Username" name="uname" required>
```
Comment (sjr-mayden) [inline login.html]: for attribute in label should match with id attribute of input * 2

File: login.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    $username = $_POST['uname'];
+    $password = $_POST['psw'];
+    session_start();
+
+    if($username == USERNAME && password_verify($password, PASSWORD)) {
+        $_SESSION['loggedIn'] = true;
+
+        header('Location: admin.php');
+    } else {
+        header('Location: index.html');
```
Comment (sjr-mayden) [inline login.php]: header should point to index.php

File: login.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
```
Comment (sjr-mayden) [inline login.php]: Redundant new line

File: login.php [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+define('PASSWORD', '$2y$10$TdIbbxTzrqo.DIgWPHw7t.Q1ZDLzzxHwPw.RBhVHVuIxL/eKoCp3O');
+
+if(!empty($_POST['uname']) && !empty($_POST['psw'])) {
+    $username = $_POST['uname'];
+    $password = $_POST['psw'];
+    session_start();
+
+    if($username == USERNAME && password_verify($password, PASSWORD)) {
+        $_SESSION['loggedIn'] = true;
+
```
Comment (sjr-mayden) [inline login.php:12]: Redundant line break

File: login.php [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+define('USERNAME', 'audrey');
+define('PASSWORD', '$2y$10$TdIbbxTzrqo.DIgWPHw7t.Q1ZDLzzxHwPw.RBhVHVuIxL/eKoCp3O');
+
+if(!empty($_POST['uname']) && !empty($_POST['psw'])) {
+    $username = $_POST['uname'];
+    $password = $_POST['psw'];
+    session_start();
+
+    if($username == USERNAME && password_verify($password, PASSWORD)) {
```
Comment (sjr-mayden) [inline login.php]: There is currently no space after if. There should be a space after the if.

File: login.php [hunk 8]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+define('USERNAME', 'audrey');
+define('PASSWORD', '$2y$10$TdIbbxTzrqo.DIgWPHw7t.Q1ZDLzzxHwPw.RBhVHVuIxL/eKoCp3O');
+
+if(!empty($_POST['uname']) && !empty($_POST['psw'])) {
```
Comment (sjr-mayden) [inline login.php]: There is currently no space after if. There should be a space after the if.

File: projectForm.php [hunk 9]
```diff
@@ -1,4 +1,8 @@
 <?php
+if(empty($_SESSION['loggedIn']) || $_SESSION['loggedIn'] = false) {
```
Comment (sjr-mayden) [inline projectForm.php:2]: There is currently no space after if. There should be a space after the if.

File: admin.php [hunk 10]
```diff
@@ -1,4 +1,9 @@
 <?php
+
+if(empty($_SESSION['loggedIn']) || $_SESSION['loggedIn'] = false) {
```
Comment (sjr-mayden) [inline admin.php:3]: There is currently no space after if. There should be a space after the if.

**Review by sjr-mayden — APPROVED** [review]

**Pull request closed without merging**